prepping for new npm release, fixing a lot of security issues (#220)
GannettDigital/simulato: CHANGELOG.md

## Pending Version

## 0.10.0

* Raghu Dantuluri and Zach Knox
  * Updated rerun logic to allow for parallelism in retries

>     * Previously, Simulato would wait until all tests of a particular run were complete before triggering a retry for needed tests. Now, teses will rerun independent of eachother.

Reviewer: ```suggestion
    * Previously, Simulato would wait until all tests of a particular run were complete before triggering a retry for needed tests. Now, tests will rerun independent of each other.
```
